src/lib/menu/menu-trigger.ts

```diff
@@ -203,7 +216,12 @@ export class MdMenuTrigger implements AfterViewInit, OnDestroy {
   openMenu(): void {
     if (!this._menuOpen) {
       this._createOverlay().attach(this._portal);
-      this._closeSubscription = this._menuClosingActions().subscribe(() => this.menu.close.emit());
+      this._closeSubscription = this._menuClosingActions().subscribe(() => {
+        if (this.menu.close) {
```

Comment:
Why do we need this null check? The `close` emitter should always be there.

Reply:
On the interface `MdMenuPanel` it declared a `close` EventEmitter. As part of this deprecation, I made that an optional interface and made `closed` the new interface. The implementation of this required I add a null check in those spots. If we are ok with removing that from the interface, we can remove all of this.